src/tools/usdr_dm_create.c

```diff
@@ -685,21 +758,21 @@ int main(UNUSED int argc, UNUSED char** argv)
     }
 
 stop:
-    usdr_dme_get_uint(dev, "/dm/debug/rxtime", temp);
     usdr_dme_get_uint(dev, "/dm/debug/rxtime", temp);
     //usdr_dme_get_uint(dev, usdr_dmd_find_entity(dev, "/dm/debug/rxtime"), temp);
 
+    //Stop RX&TX streams
     if (dorx) {
         res = usdr_dms_op(usds_rx, USDR_DMS_STOP, 0);
         if (res) {
-            fprintf(stderr, "Unable to stop data stream: errno %d\n", res);
+            USDR_LOG(LOG_TAG, USDR_LOG_ERROR, "Unable to stop data stream: errno %d", res);
```

Comment:
It should to specify the stream direction, as example `Unable to stop RX data stream`